components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json

```diff
@@ -188,5 +188,17 @@
   "braveRewardsCreatingText": {
     "message": "Creating wallet",
     "description": ""
+  },
+  "contributeNotificationError": {
+    "message": "There was a problem processing your contribution.",
+    "description": ""
```

Comment:
Please add descriptions for what the string is for.

Reply:
added